gemm/cutlass/gemm.py

```diff
@@ -0,0 +1,39 @@
+import torch
+from torch import Tensor
+
+from compile import Compile
+
+__all__ = [
+    "gemm_func",
```

Comment:
Can we provide a more precise name, such as `cutlass_gemm` or `cute_gemm`, for consistency in benchmarking in Python?

Reply:
Fixed. I would prefer to keep `gemm_func` as a general name and use the parent scope's name to distinguish specific implementations, such as:

```python
from cutlass.gemm import gemm_func as cutlass_gemm
```

The current implementations are quick experiments to diagnose performance issues and will be changed rapidly in future PRs.